users/ignoring.rst

```diff
@@ -140,3 +134,23 @@ all files and directories called "foo", ending in a "2" or starting with
   ``some/directory/`` matches the content of the directory, but not the
   directory itself. If you want the pattern to match the directory and its
   content, make sure it does not have a ``/`` at the end of the pattern.
+
+Include only
+~~~~~~~~~~~~
+
+You can write include-only ``.stignore`` files::
+
+    !/bar
+    !/baz.txt
+    !/foo
+    *
+
+This will only sync ``bar``, ``baz.txt`` and ``foo`` while ignoring everything
+else in the folder.
+
+.. note::
+
+   Syncthing optimizes the performance of this use case by only scanning and
+   watching the included files and directories. This applies **only** to top-level
+   include patterns. So having a pattern like ``!foobar`` or ``!**foo**`` without a
```

Comment:
If giving concrete examples, I think something like ``!/foo/bar`` should be present.

Reply:
Yup, after looking at the tests in your pull requests I understood the behavior better. 👍